### PR TITLE
Add reducers and actions

### PR DESCRIPTION
**Pull request by Ibnballo1**
In this project, I added reducers and actions for the books and categories file. I also added configure store

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Ibnballo, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have an appropriate ReadMe. Kindly ensure that your ReadMe is detailed and reflects your project. I suggest you use this [recommended template](https://github.com/microverseinc/readme-template/blob/master/README.md). Kindly request for another review when fix is done.


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+const bookReducer = (state = bookState, action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [
+        ...state,
+      ];
+    case REMOVE_BOOK:
+      return [
+        ...state,
+      ];
```
Comment (omarsalem7) [inline src/redux/books/books.js]: - [ ] Kindly make sure when in ADD_BOOK action to add the new book to the state and also for REMOVE_BOOK to remove the book from the state by using the filter method.

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  type: CHECK_STATUS,
+});
+
+// Reducer
+const categoryReducer = (state = categories, action) => {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return [
+        ...state,
+      ];
```
Comment (omarsalem7) [inline src/redux/categories/categories.js]: - [ ] Kindly make sure when the action CHECK_STATUS the function returns a string "Under construction".

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @Ibnballo1 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- There is no linter error present ✔️
- The Pull Request has a proper title. ✔️
- Correct git-flow ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/books.js (hunk 1 above)
Comment (Ibnballo1) [reply src/redux/books/books.js]: @omarsalem7 am I to add books to the books component?

**Review by Munsa1 — APPROVED** [review]
Hi @Ibnballo1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**